app/org/sagebionetworks/bridge/dynamodb/DynamoUploadSchema.java

```diff
@@ -71,7 +72,7 @@ public String getKey() throws InvalidEntityException {
      *
      * @see #getKey
      */
-    @JsonIgnore
+    @JsonProperty
```

Comment:
The key is the compound DynamoDB key, consisting of the study and the schema ID. I did it this way so that schema IDs could be scoped within a study, for the convenience of the researchers.